google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -67,6 +68,10 @@ def run(self):
         """Runs the import."""
 
         self.session.add(self.model)
+        self.model.add_description(json.dumps({
+            "source":"empty",
+            "prestine":True
```

Comment:
Should this be spelled as "pristine" instead?  BTW: what is the significance or usage of this field?

Reply:
If a data model is touched by a playground manipulation, pristine will be False, so scanner knows this model has been touched.
In addition, you can know the source of a data model from description and if it is from an inventory, you can tell the exact info of the source.
Since we use json for description, new items can be added into the description for future proof.